app/representers/api/musical_work_representer.rb

```diff
@@ -9,7 +9,7 @@ class Api::MusicalWorkRepresenter < Api::BaseRepresenter
   property :label
   property :album
   property :year
-  property :excerpt_length, as: :length
+  property :excerpt_length, as: :duration
```

Comment:
In both cases, there is additional work to be done and it's not just a straight database lookup.

In this case, `alias_method` will not work because there is no `excerpt_length` method until the first time it is accessed, meaning we would need to:
```ruby
def duration
  excerpt_length
end
```

`AudioFile` is set up to work the same way that this does – see [the representer](https://github.com/PRX/cms.prx.org/blob/b51f26b08e14888f7bc40a703cc537f605d9220e/app/representers/api/audio_file_representer.rb#L9). If this is wrong, then I'm not offering it as justification, but I can tell you that this was done this way for the same reasons.

This is high priority because it's currently breaking certain parts of beta. If you want me to add the method, I will do so, but I don't know that I agree it is more consistent.